plugins/svc_plugin/src/svc_plugin/svc.py

```diff
@@ -0,0 +1,33 @@
+import json
+#from models.sklearn.sklearn_model import SklearnModel
+from DashAI.back.models.tabular_classification_model import TabularClassificationModel
+from sklearn.svm import SVC as _SVC
+#from base.base_model import model_register
+#from main import model_register
+#from DashAI.back.registries.model_registry import ModelRegistry
+# from DashAI.back.registries.registration import get_model_registry
```

Comment:
To merge I think it is good that you remove all the commented code.
```suggestion
from DashAI.back.models.tabular_classification_model import TabularClassificationModel
from sklearn.svm import SVC as _SVC
```